nautobot/extras/tests/test_views.py

```diff
@@ -2294,7 +2294,7 @@ def test_required_relationships(self):
             "relationship &quot;VLANs require at least one Device&quot;",
         )
         for vlan in vlans[:3]:
-            self.assertContains(response, f"{str(vlan)}")
+            self.assertContains(response, f"{vlan!s}")
```

Comment:
It's strange that this was an f-string in the first place. What's wrong with `str(vlan)`?